assets/caniuse.css

```diff
@@ -16,6 +16,27 @@ button.caniuse-cell {
   position: relative;
 }
 
+/* handle case when printing */
+@media print {
+  .caniuse-cell.y::before {
+    content: " ✔️ ";
```

Comment:
Just wondering, do we need the white space at the start? I get why the whitespace after the emoji might be a good idea... but is the whitespace at the start not handled by the padding?

Reply:
The content value gets trimmed anyway so, I just put the spaces for it to look clearly visible in the code.

Reply:
Ah, ok. No, then let's remove the white space. I work on a browser, so I don't trust browsers :)

Reply:
Done!